glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: b2ac93355c3f551a75216a800337cee9321f6c9a04a18ab1fa8d8152e89b7595
```

Comment:
The hash is based on the content of the glide.yaml file. However, the manifest file did not change.